src/components/MarketSelector/MarketSelector.tsx

```diff
@@ -53,6 +55,7 @@ type MarketOption = {
 };
 
 export function MarketSelector({
+  chainId,
```

Comment:
Did not what to make this component dirty but really there is not reason not to. it just there were no usages of selectors here so i thought it would be good to keep it that way